product/02-innersource-and-agile.md

```diff
@@ -0,0 +1,82 @@
+# InnerSource and Agile
+
+You want to improve your product and deliver those improvements faster to customers.
+You want to make stakeholders happy.
+InnerSource helps your team deliver value and maintain autonomy especially in a highly interconnected world.
+
+
+## Autonomous teams in an interconnected world
+
+* organisations try to deliver value to customers quickly
+* to do that lengthy roadmap synchronisation settings are to be avoided
+* often those discussions stem from team dependencies: Releases waiting for the testers, deployment waiting for time in the ops team.
+As a result, cross functional teams where testing happens close to development and those who build software own running the software have proven to be successful.
+* but there is another type of inter-team dependency:
+* however we live in an interconnected world: Software written today relies on many dependencies
+* Each source code dependency, each call to some service, each external dataset used boils down to a dependency between two teams.
+* What if you need changes made to such a dependency?
+   * Traditional options:
+      * submit a feature request and wait it out
+      * build a workaround
+      * if nothing else helps, escalate your issue
+   * added benefit of InnerSource:
+      * help with the solution
+
+* But team members working in each others repositories sounds like chaos?
+   * InnerSource comes with a set of roles and processes, contributions happen in a structured way
+   * each InnerSource project has a clear set of trusted committers with clear accountabilities that go beyond simply reviewing code.
+   * support for more autonomy by being able to fix upstream artifacts without sacrifycing quality due to a clear review process
+
+* Todays world moves towards a remote first culture - InnerSource has a focus on written communication and lends itself easily to remote first.
+
+
+## Does InnerSource replace Agile?
+
+* short answer: No, not at all.
+* Instead they complement each other:
+
+
+- How InnerSource and Agile complement each other
+   - Well factored and well tested code leads to faster on-ramp for agile teams, but also helps InnerSource allowing for faster on-ramp for contributors
+   - A focus on working together and not assigning individual tasks allows for easier on-ramp of contributors.
+   - Existing local pairing can make cross team pairing and sharing of work in progress easier.
+   - Existing test driven development means that confidence in contributions is higher.
+   - XP teams have a "leave the code in better shape than you found it" attitude - mitigates the risk of decline in quality or co-hesion of code when faced with multiple contributions from different sources.
+
+## Common misunderstandings when coming from agile teams
```

Comment:
Separation of ownership for a feature - Host team ultimately owns the feature design and ongoing maintenance.  Guest team owns the code creation.

Reply:
I've tried to address that in the fifth section below.